tools/build/scanCode.py

```diff
@@ -21,6 +21,7 @@ def exceptional_paths():
     return [
         "bin/wsk",
         "bin/wskadmin",
+        "config/perryEnv.sh",
```

Comment:
Do you need to add this? I'ld rather not conflate scanCode with personal exceptions